internal/action/delete.go

```diff
@@ -39,7 +43,7 @@ func (s *Action) Delete(c *cli.Context) error {
 		}
 	}
 
-	if recursive {
+	if recursive && key == "" {
```

Comment:
This is unnecessary since it would have returned earlier otherwise, but I guess it won't hurt.